### DISCUSSION TITLE
Error: HTTP Error: Failed to download extension "mysqlscanner"

### DISCUSSION THREAD

**Discussion by fuhaiq** [duckdb/duckdb#9534] [Q&A] [answered]
I got "Access Denied" when try to "install mysqlscanner" in duckdb command line.

`Error: HTTP Error: Failed to download extension "mysqlscanner" at URL "http://extensions.duckdb.org/v0.9.1/osx_amd64/mysqlscanner.duckdb_extension.gz"`

Is there any alternative place where I can download ?

**Comment by carlopi** [comment, marked as answer]
Currently this is not properly distributed (yet).

If you feel adventurous you can get the extension artifacts from https://github.com/duckdb/duckdb_mysql/actions, then select your platform, then go to "Artifacts" to the bottom of the page, click to download, unzip, place the downloaded extension in your current folder. Then `duckdb -unsigned` (or alternative configuration option for your client, given it's currently not signed) and `LOAD ./mysql_scanner.duckdb_extension`.

**Reply by fuhaiq** [reply]
Thanks!
